install_config/adding_hosts_to_existing_cluster.adoc

```diff
@@ -309,3 +309,133 @@ $ ansible-playbook [-i /path/to/file] \
 
 . After the playbook completes successfully,
 xref:../install_config/install/advanced_install.adoc#advanced-verifying-the-installation[verify the installation].
+
+[[replacing-existing-masters]]
+== Replacing Existing Masters
```

Comment:
After reading the whole section, it appears to me it should be the case of replacing existing masters **collocated with etcd**.